src/components/internal-events/internal-events.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { getEventsForPresenter } from '../../helpers/eventHelper';
+import InternalEvent from '../internal-event/internal-event';
+
+const InternalEvents = ({ presenterName, presenterNickname }) => {
```

Comment:
I would prefer to call this component event-list. What do you think?

This way we could have consistency in the project (like profile-box and profile-list

Reply:
Agree, done